source/geod24/concurrency.d

```diff
@@ -287,7 +287,7 @@ protected:
         }
 
         if (sz == 0)
-            this.m_fibers ~= new InfoFiber(&wrap, 16 * 1024 * 1024);
+            this.m_fibers ~= new InfoFiber(&wrap);
```

Comment:
Why ?

Reply:
I wanted to make it possible to set different stack sizes for each fiber.
But this doesn't matter, so I'll put it back the old way.